### PR TITLE
Check for IntelliJ Project SDK

### PR DESCRIPTION
**Pull request by alyssaruth**
Tested locally with various setups - no SDK at all, a python SDK, the incorrect Java version - all working fine and logging sensibly :smile: 

Will plug in with an extra `.suggest_If` over in Glean to direct to where ASDF java installs live.

### CODE REVIEW

File: daktari/checks/intellij_idea.py [hunk 1]
```diff
@@ -160,3 +161,38 @@ def validate_query_result(self, result):
         current_package_manager = str(key_json["keyToString"]["nodejs_package_manager_path"])
         logging.debug(f"IntelliJ node package manager set to: {current_package_manager}")
         return current_package_manager.__contains__(self.package_manager_path)
+
+
+class IntelliJProjectSdkJavaVersion(XmlFileXPathCheck):
+    name = "intellij.jdkVersionConfigured"
+    file_path = ".idea/misc.xml"
```
Comment (mattrussell-sonocent) [inline daktari/checks/intellij_idea.py:168]: Dunno if it's too much faff, but would it be worth a unit test that looks at a real `misc.xml` that we include in the repo?

Comment (alyssaruth) [reply daktari/checks/intellij_idea.py:168]: Yeah, not a faff at all I don't think - can sort out some tests for it tomorrow :+1:

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -33,10 +35,9 @@ jobs:
       run: |
         mypy daktari
     - name: test
+      working-directory: daktari
       run: |
-        sudo apt-get update
-        sudo apt-get install -y xclip xorg
-        python -m unittest discover
+        python -m pytest
```
Comment (mattrussell-sonocent) [inline .github/workflows/build.yaml:40]: oh nice - it's been a few years, but I have a mental note that pytest is just way nicer, maybe at some point we should look to port our tests to take full advantage!

**Review by mattrussell-sonocent — APPROVED** [review]

File: .github/workflows/build.yaml (hunk 2 above)
Comment (alyssaruth) [reply .github/workflows/build.yaml:40]: yeah, the output is a lot better for starters :smile:

**Pull request merged**